src/cclib/parser/cjsonHolder.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of cclib (http://cclib.github.io), a library for parsing
+# and interpreting the results of computational chemistry packages.
+#
+# Copyright (C) 2007-2014, the cclib development team
+#
+# The library is free software, distributed under the terms of
+# the GNU Lesser General Public version 2.1 or later. You should have
+# received a copy of the license along with cclib. You can also access
+# the full license online at http://www.gnu.org/copyleft/lgpl.html.
+
+from .data import ccData
+
+class cjsonHolder:
```

Comment:
Maybe the JSON hierarchical structure can be encoded in the ccData object?
Perhaps something like "level1/level2/attribute" or
"level1:level2:attribute"?


On Tue, Jul 19, 2016 at 8:09 AM Karol M. Langner <notifications@github.com>
wrote:

> In src/cclib/parser/cjsonHolder.py
> <https://github.com/cclib/cclib/pull/288#discussion_r71357431>:
>
> > @@ -0,0 +1,148 @@
> > +# -*- coding: utf-8 -*-
> > +#
> > +# This file is part of cclib (http://cclib.github.io), a library for parsing
> > +# and interpreting the results of computational chemistry packages.
> > +#
> > +# Copyright (C) 2007-2014, the cclib development team
> > +#
> > +# The library is free software, distributed under the terms of
> > +# the GNU Lesser General Public version 2.1 or later. You should have
> > +# received a copy of the license along with cclib. You can also access
> > +# the full license online at http://www.gnu.org/copyleft/lgpl.html.
> > +
> > +from .data import ccData
> > +
> > +class cjsonHolder:
>
> That's true! I forgot about that. We can reuse that, but the JSON has a
> hierarchical structure which requires some more information, right?
> Currently the code implements this.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/cclib/cclib/pull/288/files/e182d4d0b4675cbc448ea069250db8375e191769#r71357431>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACyd38ZfQnBTAyaT6JLFgey3eTpAWaueks5qXOgKgaJpZM4JOR_J>
> .
>